tests/test_methods_nai.py

```diff
@@ -0,0 +1,71 @@
+"""This test aims to compare to previous PDA and PDAW results that were published and compared to exact QD.
+We use NaI as an example with two different laser pulses. If this test fails, the results produced by the code are unreliable!"""
+
+from pathlib import Path
+
+import numpy as np
+
+from promdens.promdens import InitialConditions, LaserPulse
+
+# path to the tests folder
+path = Path(__file__).parent.absolute()
+
+def test_pdaw_nai(tmp_path):
+    ics = InitialConditions(nsamples=1000, nstates=1, input_type='file')
+
+    ics.read_input_data(fname=path/'NaI_reference/test_data_nai.dat', energy_unit='a.u.', tdm_unit='a.u.')
+
+    fstoau = 41.341374575751
+    pulse = LaserPulse(omega=0.13520905, fwhm=20*fstoau, envelope_type='gauss', lchirp=0, t0=0)
+
+    ics.calc_field(pulse=pulse)
+
+    ics.windowing()
+
+    # moving output to a temporary path dedicated to this test
+    tmp_folder = tmp_path / "pdaw"
```

Comment:
`tmp_path` and is unique for this test so you can just use it instead of creating a subfolder. (same for the other test)